src/main/java/quotes/App.java

```diff
@@ -4,22 +4,31 @@
 package quotes;
 
 import com.google.gson.Gson;
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileReader;
+
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
 
 public class App {
 
     public static void main(String[] args) throws FileNotFoundException {
+        try {
+            //Add the joke to the quotesArray
+            FileWriter jokeWriter = new FileWriter("src/main/resources/apijoketest.json");
+            Gson gson = new Gson();
+            Quote joke = new Quote("The Universe", new App().getDataFromAPI());
+            jokeWriter.append(gson.toJson(joke));
```

Comment:
https://www.geeksforgeeks.org/java-appending-string-file/ Here is a resource that shows how to use FileWriter and BufferedWriter to add a string to a file, if you want to continue pursuing this.

However, appending to the file would not append to the array, it doesn't know that the thing in the file is an array, because it is json.